src/python/WMCore/GlobalWorkQueue/CherryPyThreads/MonitoringTask.py

```diff
@@ -4,11 +4,12 @@
 from WMCore.WorkQueue.WorkQueue import globalQueue
 from WMCore.Services.WMStats.WMStatsWriter import WMStatsWriter, convertToServiceCouchDoc
 
+GQ_THREADS = ['CleanUpTask', 'LocationUpdateTask', 'ReqMgrInteractionTask', 'MonitoringTask']
```

Comment:
I want to eventually have the same as other application HeartbeatMonitor but this one has additional information attached so GQ has a little different structure now.
The reason I explicitly  has the thread list instead of getting from config is that there might be the case we don't want to monitor the thread. (i.e. Heartbeat thread itself or thread running in multiple machines) but I think it wouldn't hurt to monitor everything.  I think it is good idea to get them from config. I will change that.